src/main.py

```diff
@@ -0,0 +1,102 @@
+from AlertsAggregator import AlertsAggregator
+from OrefAlertsRetrieve import OrefAlertsRetrieveData  
+from fastapi import FastAPI, HTTPException 
+import pandas as pd
+import requests
+from Errors import RetrieveDataException, WrongSettlementException
+from models import AlertsQueryInput
+from datetime import datetime, timedelta
+app = FastAPI()
+
+CSV_FILE_PATH = "C:/Projects/Python/Projects/BestTimeToShower/data/data.csv"
```

Comment:
use .env file for this PATH.

see that comment:

https://github.com/YamAlmog/BestTimeToShower/pull/3#discussion_r1412887421